### PR TITLE
Implemented basic player movement

### PR DESCRIPTION
**Pull request by CerberusOne**
-Implemented basic player movement to fly up, down, right and left. We have to make the movement more "smooth" in the future.
-Implemented very basic cave environment with obstacles. 
-Player can't move through a obstacle (just for testing, change to "death" state in the future)
-Player can still move through cave walls (again, change to "death" in future)
-There is a function called OnCollisionEnter inside the playerMovement script attached to the player object. Update this function for player collision events with obstacles and walls.

Also FYI, if replacing this project in the future, make sure you update the Editor of the new project so you can commit it to a git repo:
-Edit > Project Settings > Editor 
-Change Version Control > Mode to "Visible Meta Files"
-Change Asset Serialization > Mode to "Force Text"

### CODE REVIEW

File: Cave Flier/Assets/Scripts/cameraMovement.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class cameraMovement : MonoBehaviour {
```
Comment (jacobfrank4) [inline Cave Flier/Assets/Scripts/cameraMovement.cs]: brace should be on new line

File: Cave Flier/Assets/Scripts/cameraMovement.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    public GameObject player;
+    private Vector3 offset;
+
+	// Use this for initialization
+	void Start () {
```
Comment (jacobfrank4) [inline Cave Flier/Assets/Scripts/cameraMovement.cs]: brace should on new line

File: Cave Flier/Assets/Scripts/playerMovement.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+using UnityEngine;
+using System;
+using System.Globalization;
+
+
+public class playerMovement : MonoBehaviour {
```
Comment (jacobfrank4) [inline Cave Flier/Assets/Scripts/playerMovement.cs]: brace should be on new line

File: Cave Flier/Assets/Scripts/playerMovement.cs [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    public float speed = 6;
+
+	// Use this for initialization, called on first frame the object is active
+	void Start () {
```
Comment (jacobfrank4) [inline Cave Flier/Assets/Scripts/playerMovement.cs]: brace should be on new line

**Review by jacobfrank4 — CHANGES_REQUESTED** [review]

File: Cave Flier/Assets/Scripts/cameraMovement.cs (hunk 1 above)
Comment (CerberusOne) [reply Cave Flier/Assets/Scripts/cameraMovement.cs]: file not needed anymore, deleted

File: Cave Flier/Assets/Scripts/playerMovement.cs [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+using System;
+using System.Globalization;
+
+
+public class playerMovement : MonoBehaviour
+{
+    public float speed = 6;
+
+	// Use this for initialization, called on first frame the object is active
+	void Start ()
```
Comment (jacobfrank4) [inline Cave Flier/Assets/Scripts/playerMovement.cs]: incorrect spacing.  the function call and comment are further indented than the opening brace.

**Review by jacobfrank4 — CHANGES_REQUESTED** [review]

**Review by jacobfrank4 — APPROVED** [review]

**Review by jacobfrank4 — APPROVED** [review]

**Review by jacobfrank4 — APPROVED** [review]

**Review by jacobfrank4 — APPROVED** [review]

**Pull request merged**